sensor_msgs/RegionOfInterest roi
geometry_msgs/Point center3d